torch/_dynamo/utils.py

```diff
@@ -934,6 +934,11 @@ def same(
             if not exact_dtype:
                 ref = ref.to(res.dtype)
 
+            if ref.is_nested and res.is_nested:
+                return same(ref.unbind(), res.unbind())
+            elif ref.is_nested or res.is_nested:
```

Comment:
We do have some ops that do mixed operations

Reply:
This is used in cross-ref tests where we want to get the same result with (`res`) and without (`ref`) an enabled `FakeTensorMode`

Reply:
That's not what's tested here, this is just saying that both the reference (regular cpu nested tensor) and the test (meta nested tensor) have to always agree on the output being nested or not.